test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -88,7 +88,7 @@ public void SdcaRegressionNameCollision()
                 separator: ';', hasHeader: true);
 
             var est = reader.MakeNewEstimator()
-                .Append(r => (r.label, r.Score, score: ctx.Trainers.Sdca(r.label, r.features, maxIterations: 2, advancedSettings: s => s.NumThreads = 1)));
```

Comment:
>.NumThreads = 1 [](start = 136, length = 15)

Same here.

Reply:
fixed

---
In reply to: [248104199](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248104199) [](ancestors = 248104199)